packages/http-specs/package.json

```diff
@@ -8,7 +8,7 @@
     "watch": "tsc -p ./tsconfig.build.json --watch",
     "build": "tsc -p ./tsconfig.build.json",
     "clean": "rimraf dist/ temp/",
-    "test:e2e": "pnpm validate-scenarios  && pnpm validate-mock-apis && pnpm validate-client-server",
+    "test:e2e": "pnpm build && pnpm regen-docs && pnpm validate-scenarios  && pnpm validate-mock-apis && pnpm validate-client-server",
```

Comment:
please revert that, we shouldn't rebuild again

Reply:
I just want to make it convenient for devs to check the test case with one command. I will revert this line and add a new command.